src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/34.diff.sql

```diff
@@ -31,8 +31,8 @@ CREATE OR ALTER PROCEDURE dbo.GetInstanceBatchesByTimeStamp
     @batchSize INT,
     @batchCount INT,
     @status TINYINT,
-    @startTimeStamp DATETIME,
-    @endTimeStamp DATETIME,
+    @startTimeStamp DATETIMEOFFSET(0),
```

Comment:
Do you need any more granularity than this?

Reply:
I don't think so, I already gave plus or minus one day. So the dates are not that exact.